File: solutions/cpp/darts/1/darts.cpp
#include "darts.h"
namespace darts {
// TODO: add your solution here
int score(double x,double y)
{
    double r2 = x * x + y * y;
    if (r2 <= 1) return 10;      
    if (r2 <= 25) return 5;      
    if (r2 <= 100) return 1; 
    return 0;
}
}  // namespace darts

File: solutions/csharp/darts/1/Darts.cs
public static class Darts
{
    public static int Score(double x, double y)
    {
      double r2 = x*x + y*y;
        if(r2 <= 1) return 10;
        if(r2 <= 25) return 5;
        if(r2 <= 100) return 1;
        return 0;
    }
}
